config/webpack.common.config.js

```diff
@@ -4,6 +4,7 @@ const path = require('path');
 
 module.exports = {
   entry: {
+    new_relic: path.resolve(__dirname, '../src/newRelic.js'),
```

Comment:
this sort of is the defacto prod behavior now since dev overrides the entry... if we want to move it that's ok with me. i think we need segment and new relic in prod and stage, and i think it's good to keep segment in dev because we are likely to develop more eventing in the future, but not likely to do much else for new relic.

Reply:
I had missed Dev overrides this, thanks for clarifying!